CHANGELOG

```diff
@@ -1,3 +1,9 @@
+* Wed Jul 28 2021 Andy Adrian <andy.adrian@onyxpoint.com> - 8.2.1
```

Comment:
IMHO this is a new feature and not a bug fix.  So it should be a 'Y' version bump, i.e '8.3.0'

Reply:
Correction:  Since version 8.2.0 has not been released, the version can be left at 8.2.0.